### PR TITLE
Ledger fixes

### PR DESCRIPTION
**Pull request by shjortConcordium**
## Purpose

Fix stuff based on feedback from ledger + cleanup unnecessary flows.

## Changes

- Add accept/reject in “compare” screen
- Add CCD/unit to transfers
- Merge flows in identity issuance
- Remove chain updates
- Remove old baker flows
- Fixed tests

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

**Review by bisgardo — CHANGES_REQUESTED** [review]
Please remove the `rust_tests` that are no longer relevant.

**Comment by bisgardo** [comment]
It's not that big of a deal in this PR but I think that in general we should strive to separate formatting changes and other trivialities (like bulk deletion) from more complex changes.

File: src/signPublicInformationForIp.c [hunk 1]
```diff
@@ -7,15 +7,15 @@
...
 static tx_state_t *tx_state = &global_tx_state;
 
-UX_STEP_CB(ux_sign_public_info_review, nn, sendSuccessNoIdle(), {"Review identity", "provider info"});
-UX_FLOW(ux_review_public_info_for_ip, &ux_sign_public_info_review);
-
 UX_STEP_CB(
-    ux_sign_public_info_for_i_public_key_0_step,
+    ux_sign_public_info_for_ip_public_key_0_step,
     bnnn_paging,
     sendSuccessNoIdle(),
```
[the remark this answers is not in the document]
Comment (shjortConcordium) [reply src/signPublicInformationForIp.c:13]: I checked at least with our other flows, and there we have a "continue" screen, so I've added that.

File: src/signPublicInformationForIp.c [hunk 2]
```diff
@@ -34,7 +34,15 @@ UX_STEP_NOCB(
     bn,
     {"Signature threshold", (char *) global.signPublicInformationForIp.threshold});
 UX_FLOW(
-    ux_sign_public_info_for_ip_threshold,
+    ux_sign_public_info_for_ip_final,
+    &ux_sign_public_info_for_ip_public_key_0_step,
```
[the remark this answers is not in the document]
Comment (shjortConcordium) [reply src/signPublicInformationForIp.c:38]: Yeah, good catch. I should've created a new step without the callback, but with the continue screen, we don't a version with callback. 
(I changed the name to `ux_sign_public_info_for_ip_display_public_key`)
(Also renamed the one for threshold to `ux_sign_public_info_for_ip_display_threshold`)

File: src/getPublicKey.c [hunk 3]
```diff
@@ -11,24 +11,30 @@ static tx_state_t *tx_state = &global_tx_state;
...
 
+UX_STEP_VALID(ux_decline_step, pb, sendUserRejection(), {&C_icon_crossmark, "Decline"});
+
 // UI definitions for the approval of the generation of a public-key. This prompts the user to accept that
 // a public-key will be generated and returned to the computer.
 UX_STEP_VALID(
     ux_generate_public_flow_0_step,
     pnn,
     sendPublicKey(true),
     {&C_icon_validate_14, "Public key", (char *) global.exportPublicKeyContext.display});
```
[the remark this answers is not in the document]
Comment (shjortConcordium) [reply src/getPublicKey.c:22]: Yeah, the display is a bit "compressed", but it should be clear when used together with the wallet.
We could add an issue for this, but I don't think we need to fix it here. :thinking:

**Review by bisgardo — APPROVED** [review]

**Pull request merged**